src/main/kotlin/uk/gov/justice/digital/hmpps/prisonerfromnomismigration/alerts/AlertsByPrisonerMigrationService.kt

```diff
@@ -91,9 +91,7 @@ class AlertsByPrisonerMigrationService(
       telemetryClient.trackEvent(
         "alerts-migration-entity-migrated",
         mapOf(
-          "latestNomisBookingId" to latestNomisBookingId,
           "offenderNo" to offenderNo,
-          "status" to status,
           "migrationId" to context.migrationId,
```

Comment:
these fields have been dropped from endpoint for performance reasons  - they were used only for analytics